Source/Components/ImageGlass.Heart/Img.cs

```diff
@@ -50,7 +51,13 @@ public class Img: IDisposable
         /// <summary>
         /// Gets, sets Bitmap data
         /// </summary>
-        public Bitmap Image { get; set; } = null;
+        public Bitmap Image { get; private set; } = null;
+
+
+        /// <summary>
+        /// Gets, sets original MagickImage data
+        /// </summary>
+        public IMagickImage OriginalImage { get; private set; } = null;
```

Comment:
Storing `IMagickImage` is extremely costly.
My suggestion when we need to modify/save the viewing file:
- just read the file again,
- make change,
- and save

This should be done in `igcmd.exe`, because for those heavy files, it may take some time to complete, and the UI thread will be blocked in the meanwhile.

![image](https://user-images.githubusercontent.com/3154213/75086517-4759ff80-5570-11ea-922c-6de67c8a342d.png)

Reply:
Do I understand you correctly that starting new relatively heavy process, reading a file again and storing & passing around all changes (which complicates the source code) is more efficient than extra ~200 Mb of memory footprint?

Reply:
@TheAppX 
Yes, when saving a file, we call a function to read the file, do some manipulation, then save. It will take a bit long while saving the file, but it saves the runtime memory. If user sets the ImageBooster value to 10, the memory can go up to few GB!

Reply:
Also welcome another better idea :)